core/domain/prod_validators.py

```diff
@@ -2147,15 +2150,12 @@ def _validate_score_category(
                         'doesn\'t exist' % (
                             item.id, item.target_type,
                             model_id, model_class.__name__, model_id))
-                    continue
-                if target_model.category != score_category_sub_type:
-                    cls._add_error(
-                        'score category sub%s' % (
-                            base_model_validators.ERROR_CATEGORY_TYPE_CHECK),
-                        'Entity id %s: score category sub %s does not match'
-                        ' target exploration category %s' % (
-                            item.id, score_category_sub_type,
-                            target_model.category))
+
+                # Do not validate the subtype of score_category with the
```

Comment:
Replace with:

"Note: An exploration's category can be changed after the suggestion is submitted. Since this operation does not update the suggestion's category, we cannot assume that the exploration category matches the suggestion score category, and thus do not validate it here."

(The current comment sounds a bit more like an instruction, since you're using an imperative.)